### PR TITLE
add contour plot to aimec

### PR DESCRIPTION
**Pull request by awirb**

### CODE REVIEW

File: avaframe/out3Plot/outAIMEC.py [hunk 1]
```diff
@@ -861,3 +862,136 @@ def resultVisu(cfgSetup, inputsDF, pathDict, cfgFlags, rasterTransfo, resAnalysi
...
+            updat. dictionary with x, y coordinates for each sim contour line
+    """
+
+    # fetch input data
+    s = rasterTransfo['s']
+    l = rasterTransfo['l']
+    compData = inputs['compData']
+    simName = inputs['simName']
+
+    L, S = np.meshgrid(l, s)
```
Comment (matthiasto) [inline avaframe/out3Plot/outAIMEC.py:893]: do not use the same letters but gridL gridS for example

File: avaframe/ana3AIMEC/ana3AIMEC.py [hunk 2]
```diff
@@ -209,6 +216,9 @@ def postProcessAIMEC(cfg, rasterTransfo, pathDict, resAnalysisDF, newRasters, ti
                     growth index
             -growthGrad: float
                     growth gradient
+    contourDict: dict
+        dictionary with one key per sim and its x, y coordinates for contour line of runoutresType
+        for thresholdValue
```
Comment (fso42) [inline avaframe/ana3AIMEC/ana3AIMEC.py:221]: Maybe add what was added to the contourDict? VS on the input side

File: avaframe/ana4Stats/probAna.py [hunk 3]
```diff
@@ -261,8 +263,14 @@ def probAnalysis(avaDir, cfg, module, parametersDict='', inputDir=''):
...
                 fileName = peakFilesDF['files'][m]
-                data = np.loadtxt(fileName, skiprows=6)
                 dataLim = np.zeros((nRows, nCols))
+                fileData = IOf.readRaster(fileName)
+                data = np.flipud(fileData['rasterData'])
+
+                # fetch contourline info
+                xGrid, yGrid, _, _ = gT.makeCoordGridFromHeader(fileData['header'])
+                x, y = pU.fetchContourCoords(xGrid, yGrid, data, float(cfg['GENERAL']['peakLim']))
+                contourDict[fileName.stem] = {'x': x, 'y': y}
```
Comment (fso42) [inline avaframe/ana4Stats/probAna.py:273]: Did you check whether this slows the function down? sometimes contour generation can take quite some time...

**Review by fso42 — CHANGES_REQUESTED** [review]

File: avaframe/ana4Stats/probAna.py (hunk 3 above)
Comment (fso42) [reply avaframe/ana4Stats/probAna.py:273]: I tried. since it is only on contour per sim, it is not slowing it down

**Review by fso42 — APPROVED** [review]

**Pull request merged**